src/test/groovy/io/codearte/jfairy/producer/person/PersonSpec.groovy

```diff
@@ -214,5 +231,118 @@ class PersonSpec extends Specification {
 			person.passportNumber()
 	}
 
+	def "withFirstName should create person with specific first name"() {
+		when:
+			def person = fairy.person(withFirstName("Specificfirstname"))
```

Comment:
I know this is quite a subjective and opinionated approach when it comes to those `def`, but as in the Codearte team we all seem to share it, we would like to keep it this way.